cmd/dep/signals.go

```diff
@@ -0,0 +1,23 @@
+package main
```

Comment:
I'd move `Release()` into the interface, but the interface lives in `gps`. Maybe this should just wait for #300 to get resolved? I'm not sure.